src/index/IndexImpl.cpp

```diff
@@ -0,0 +1,1772 @@
+// Copyright 2014, University of Freiburg,
+// Chair of Algorithms and Data Structures.
+// Author: Björn Buchhold (buchhold@informatik.uni-freiburg.de)
+
+#include "./IndexImpl.h"
```

Comment:
This is the Original `Index.cpp` file renamed to `IndexImpl.cpp`
Unfortunately github did not recognize this rename.

I have marked some places, where actual change happened.